### PR TITLE
Fosterful Rebranding

### PR DESCRIPTION
**Pull request by benjaminwood**
Implement Fosterful rebranding

* Redirect app.officemomsanddads.com -> app.fosterful.com
* Update URL's around the app
* Update logo
* Update brand colors
* (unrelated bonus) active/deactivated scopes for users in admin

TODO (sometime after this PR): Update Favicon

### CODE REVIEW

**Comment by benjaminwood** [comment]
This PR is on staging now for review: https://app-staging.fosterful.org

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Office Moms & Dads’ 5-year vision is to establish and sustain sites throughout
 
 ## What does the scheduler do?
 
-It is a Ruby/Rails based app that allows the coordination of scheduling between various workers and volunteers supporting children entering the foster care system. Specifically, it is designed to alert volunteers when there is need of a kind person to sit with children during a scary time, often on short notice. For more information about the organization, please visit [Office Moms and Dads](http://www.officemomsanddads.com).
+It is a Ruby/Rails based app that allows the coordination of scheduling between various workers and volunteers supporting children entering the foster care system. Specifically, it is designed to alert volunteers when there is need of a kind person to sit with children during a scary time, often on short notice. For more information about the organization, please visit [Office Moms and Dads](http://www.fosterful.org).
```
Comment (nvick) [inline README.md:28]: The link content needs to be updated

File: README.md [hunk 2]
```diff
@@ -226,9 +226,9 @@ For all other questions and concerns related to this project, please open an iss
 
 ## Support and Contact details
 
-Interested in finding out more about what we do, or how you can support our mission? Please visit [https://officemomsanddads.com](https://officemomsanddads.com).
+Interested in finding out more about what we do, or how you can support our mission? Please visit [https://fosterful.org](https://fosterful.org).
 
-If after reading about us, you still have questions, you can directly [contact us](https://officemomsanddads.com/contact-us/) through our website.
+If after reading about us, you still have questions, you can [contact Us](mailto:info@fosterful.org) directly via email.
```
Comment (nvick) [inline README.md:231]: Do we want to capitalize the U in contact Us?

**Review by nvick — APPROVED** [review]
Couple comments otherwise good to go

**Comment by benjaminwood** [comment]
Thanks @nvick I've fixed those and a few other references. Deploying to staging now.

**Pull request merged**